developer-docs-site/static/examples/python/first_nft.py

```diff
@@ -152,8 +152,8 @@ def get_token_id(self, creator: str, collection_name: str, token_name: str) -> i
     print(f"Alice: {alice.address()}")
     print(f"Bob: {bob.address()}")
 
-    faucet_client.fund_account(alice.pub_key(), 10_000_000)
-    faucet_client.fund_account(bob.pub_key(), 10_000_000)
+    faucet_client.fund_account(alice.address(), 10_000_000)
```

Comment:
seems @CapCap  also has similar changes covered here https://github.com/aptos-labs/aptos-core/pull/372/files#diff-ac78e1760da5c3e43bd9914f36a1415824b276539abdb10dd1c00f9e3dc3e2e0.  we also need to make sure the address has "0x" as prefix.